batch/batch/utils.py

```diff
@@ -83,7 +101,7 @@ def worker_memory_per_core_gb(worker_type):
 
 def worker_memory_per_core_bytes(worker_type):
     m = worker_memory_per_core_gb(worker_type)
-    return int(m * 1000**3)  # GCE memory/core are in GB not GiB
+    return int(m * 1024**3)
```

Comment:
I was trying to figure out why the old method of computing cost wasn't aligning up with the new way. So one hypothesis was I was using powers of 1000 instead of 1024 and must have forgotten to change it back.

I went back and tried to figure out what the right answer is and I think it should be powers of 1000 for memory, but I'd appreciate you double checking. I'll try and take a look at disk separately.

```
jigold@batch-worker-jigold-5utdq:~$ free -b
              total        used        free      shared  buff/cache   available
Mem:     3869945856   427466752   416616448     1286144  3025862656  3213860864
Swap:             0           0           0
```

If you divide 3869945856 / 1024**3, you get 3.6Gi
If you divide 3869945856 / 1000**3, you get 3.87GB

The RAM we're supposed to have available is 3.75GB. I couldn't find any documentation anywhere whether that was supposed to be in GB or Gi. So I assumed it had to be in GB based on above.

I tried a different way to get this answer:

```
jigold@batch-worker-jigold-5utdq:~$ grep 'MemTotal' /proc/meminfo
MemTotal:        3779244 kB
```

3779244 / 1000 / 1000 = 3.78
3779244 / 1024 / 1024 = 3.60

The --si option in free is supposed to print in powers of 1000

```
jigold@batch-worker-jigold-5utdq:~$ free --si
              total        used        free      shared  buff/cache   available
Mem:        3779244      413400      410276        1256     2955568     3142584
Swap:             0           0           0
```

So this is where I got the powers of 1000 is probably the correct answer and we can say the instance has 3.75GB of RAM.

Reply:
I confirmed disk size is in GiB.

```
jigold@batch-worker-jigold-5utdq:~$ sudo fdisk -l | grep Disk
Disk /dev/loop0: 66.5 MiB, 69758976 bytes, 136248 sectors
Disk /dev/loop1: 89.1 MiB, 93454336 bytes, 182528 sectors
Disk /dev/loop2: 93.9 MiB, 98484224 bytes, 192352 sectors
Disk /dev/loop3: 55 MiB, 57618432 bytes, 112536 sectors
Disk /dev/loop4: 98.7 MiB, 103432192 bytes, 202016 sectors
Disk /dev/sda: 20 GiB, 21474836480 bytes, 41943040 sectors
Disklabel type: gpt
Disk identifier: EE3550B4-9533-4B5A-8E61-DE881FE8AF66
Disk /dev/nvme0n1: 375 GiB, 402653184000 bytes, 98304000 sectors
```